src/Presentation/GraphQL/Base/GraphSchema.cs

```diff
@@ -0,0 +1,17 @@
+using GraphQL.Types;
+
+namespace Presentation.GraphQL.Base
+{
+    public class GraphSchema : Schema
```

Comment:
Would the name `Schema` be more relevant for this class?

Reply:
Done